msteams-platform/task-modules-and-cards/cards-and-task-modules.md

```diff
@@ -25,6 +25,8 @@ The following types of cards are supported in bots for Teams:
 
 You can format card text using a subset of XML or HTML formatting or Markdown, depending on the card type. You can add and respond to card actions that open a URL, send messages and payload to the bot, or initiate OAuth flow.
 
+[People Picker in Adaptive Cards](~/task-modules-and-cards/cards/people-picker-adaptive-card.md) help to search, select, reassign, and preselect users within chat  or channel. Universal Actions for Adaptive Cards enhances user's experience by using bot as the common backend for handling actions and introduces a new action type, `Action.Execute`.
```

Comment:
Why the mention of `Universal Actions` here in People picker?

Reply:
@sowrabh-msft , 'Universal actions' sentence is existing one. Only People Picker related info has been added newly. Being a overview page, it has some introduction of all pages under cards and task modules.

Reply:
https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards-and-task-modules - `Universal Actions` sentence is not present in the current page. 
Would be good to separate the points into different paragraphs as they are unrelated features.